Buildings/Controls/OBC/Utilities/PIDWithAutotuning/AutoTuner/AMIGO/PI.mo

```diff
@@ -0,0 +1,70 @@
+within Buildings.Controls.OBC.Utilities.PIDWithAutotuning.AutoTuner.Amigo;
+block PI "Identifies the parameters of a PI controller"
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput kp(min=1E-6)
+    "Connector for the signal of the gain of a first order time-delayed model"
+    annotation (Placement(transformation(extent={{-140,40},{-100,80}}),
+        iconTransformation(extent={{-140,40},{-100,80}})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput T(min=0)
+    "Connector for the signal of the time constant of a first order time-delayed model"
+    annotation (Placement(transformation(extent={{-140,-20},{-100,20}}),
+        iconTransformation(extent={{-140,-20},{-100,20}})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput L(min=1E-6)
+    "Connector for the signal of the time delay of a first order time-delayed model"
+    annotation (Placement(transformation(extent={{-140,-80},{-100,-40}}),
+        iconTransformation(extent={{-140,-80},{-100,-40}})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealOutput k
+    "Connector for control gain signal"
+    annotation (Placement(transformation(extent={{100,50},{120,70}})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealOutput Ti
+    "Connector for time constant signal for the integral term"
```

Comment:
"Time constant of the integral term"

Reply:
No need to use "Connector for ...", or "signal for ...". Please revise the comments of other connectors.

Reply:
ditto